packages/create-fuels/src/cli.ts

```diff
@@ -130,29 +130,27 @@ export const runScaffoldCli = async ({
 }) => {
   program.parse(args);
 
-  const projectPath = program.args[0] ?? (await promptForProjectPath());
+  let projectPath = program.args[0] ?? (await promptForProjectPath());
+
   const verboseEnabled = program.opts().verbose ?? false;
 
-  if (existsSync(projectPath)) {
-    throw new Error(
-      `A folder already exists at ${projectPath}. Please choose a different project name.`
+  while (existsSync(projectPath)) {
+    log(
+      chalk.red(
+        `A folder already exists at ${projectPath}. Please choose a different project name.`
+      )
     );
-  }
 
-  if (!projectPath) {
-    throw new Error('Please specify a project directory.');
+    projectPath = await promptForProjectPath();
   }
 
-  const cliPackageManagerChoices = {
-    pnpm: program.opts().pnpm,
-    npm: program.opts().npm,
-  };
-  if (Object.values(cliPackageManagerChoices).filter(Boolean).length > 1) {
```

Comment:
I'm not sure how a user could arrive at this state given the choices are done using arrow keys, maybe I am missing something?

Reply:
It's useful when testing as a default, we could probably refactor it to be passed in but I'll leave it for now